examples/extract_xml_field_formats.py

```diff
@@ -123,7 +117,15 @@ def read_xml_field_formats(instrument, filename, verbose=False):
 
 
 def show_xml_field_formats(xml_formats, reference_instrument_name='NIRISS', out_dir=None):
+    """Print field formats to screen and write to file.
+
+    Parameters
```

Comment:
It's a good practice to include the type (str, float, etc) and description for each of the parameters if you’re going to list them

Reply:
Done.